repositories.maven.bzl

```diff
@@ -65,7 +65,13 @@ COPYBARA_MAVEN_ARTIFACT_ADDITIONAL_REPOSITORIES = [
 def copybara_maven_repositories():
     maybe(
         maven_install,
-        name = DEFAULT_REPOSITORY_NAME,
+        name = "copybara_maven",
+        artifacts = COPYBARA_MAVEN_ARTIFACTS,
+        repositories = COPYBARA_MAVEN_ARTIFACT_ADDITIONAL_REPOSITORIES + ["https://repo1.maven.org/maven2"],
+    )
+    maybe(
+        maven_install,
+        name = "maven",
```

Comment:
for bzlmod it's better to have a unique name for a repository that might be a dependency of something else, but for building bazel we have to use this name as well, right now these are dup'd but in bzlmod they have a separate set of deps